tide_site.module

```diff
@@ -588,6 +588,40 @@ function tide_site_form_alter(array &$form, FormStateInterface $form_state, $for
       }
     }
   }
+  // Add site prefix to redirect URL.
+  if ('redirect_redirect_form' === $form_id) {
+    $sites_term = 'sites';
```

Comment:
I'd rename this variable as `$vocabulary` or `$taxonomy` for clarity. It's not a "term" so could be a bit confusing or unclear when we read the code. This variable contains the name of the Vocabulary or Taxonomy we want to load from.

Alternatively, don't even store this in a variable at all. If we're not using it more than once, we don't need to use the memory allocation of a variable at all.